packages/edit-post/src/components/header/fullscreen-mode-close/index.js

```diff
@@ -15,6 +14,11 @@ import { wordpress } from '@wordpress/icons';
 import { store as editorStore } from '@wordpress/editor';
 import { store as coreStore } from '@wordpress/core-data';
 
+/**
+ * Internal dependencies
+ */
+import { __unstableMotion as motion } from '../../../../../components/src/animation';
```

Comment:
It'd be nice if there was a neater way to import this.

Reply:
There is!